Dockerfile-buildkit

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.16-alpine AS package
```

Comment:
nit pick: @heronrs could you rename it to `Dockerfile.buildkit` so IntelliSense and code formatted understand that this is a Dockerfile?